sigar_linux.go

```diff
@@ -337,6 +337,10 @@ func parseCpuStat(self *Cpu, line string) error {
 	self.Irq, _ = strtoull(fields[6])
 	self.SoftIrq, _ = strtoull(fields[7])
 	self.Stolen, _ = strtoull(fields[8])
+	/* Guest was added in 2.6, not available on all kernels */
+	if len(fields) > 9 {
```

Comment:
Worth adding a test for these two scenarios.  Can help out if we have trouble finding a pre-2.6 kernel as a source.